include/readout/ReadoutTypes.hpp

```diff
@@ -62,6 +62,8 @@ struct WIB_SUPERCHUNK_STRUCT
     return this->get_timestamp() < other.get_timestamp();
   }
 
+  bool operator==(const WIB_SUPERCHUNK_STRUCT& other) const { return this->get_timestamp() == other.get_timestamp(); }
```

Comment:
From link0 and link1, this won't be true. If we really want to make equal operator work between superchunks, we need to ensure that all 12 frames and crate/slot/fiber IDs are equal.

Reply:
If DataLinkHandlers are always separated on these composite keys, then yes, this is fine. But if we keep a fully data driven approach, in theory, the timestamp match is not sufficiently providing equality between two superchunks.